aiohttp/helpers.py

```diff
@@ -237,6 +237,7 @@ class MimeType:
     parameters = attr.ib(type=MultiDict)  # type: MultiDict[str]
 
 
+@functools.lru_cache(maxsize=56)
```

Comment:
Nice idea. But why 56?

Reply:
No particular reason. It's potentially controlled by an attacker, so it cannot be too high. Could make it 100?

Reply:
There 75 mime types in nginx default mime.types file. Quite much of them are not common / popular. I'm pretty sure that 10 will be more than enough.